plugins/dummy/DummyPortTest.cpp

```diff
@@ -662,7 +662,22 @@ void DummyPortTest::testParamDescription() {
   const string description("Code Version");
   size_t str_len = std::min(sizeof(param_description.description),
                             description.size());
+/* Some versions of GCC 9 claim that the strncpy overflows the target
```

Comment:
Nit, we generally have the /* clear at the start of a block comment, with the comment starting on the next line.

Reply:
Done.